src/main/java/org/broadinstitute/consent/http/mail/message/NewDAAUploadResearcherMessage.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.consent.http.mail.message;
+
+import com.sendgrid.helpers.mail.Mail;
+import java.io.Writer;
+import javax.mail.MessagingException;
+
+public class NewDAAUploadResearcherMessage extends MailMessage {
+  private final String NEW_DAA_UPLOAD = "New DAA uploaded for DAC in DUOS";
```

Comment:
```suggestion
  private final String NEW_DAA_UPLOAD = "New DAA uploaded and sent to researcher for DAC in DUOS";
```

Perhaps a message like this.